docs/secret_management.md

```diff
@@ -21,12 +25,66 @@ spec:
   # ...
 ```
 
-The credentials provided here will be stored in a Secret created by the Operator. By properly setting the `RBAC` on the `DatadogAgent` CRD, one can limit who is able to see those credentials.
+The credentials provided here are stored in a Secret created by the Operator. By properly setting the RBAC on the `DatadogAgent` CRD, you can limit who is able to see those credentials.
+
+### Use Secret references
+
+1. Create your Secrets:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: datadog-api-secret
+   data:
+     api_key: <api-key>
```

Comment:
should we be consistent in the placeholder style? above we have `<DATADOG_API_KEY>`